config/johnnyfive.conf-TEMPLATE

```diff
@@ -3,3 +3,14 @@ apiKey = your
 apiSecret = stuff
 tokenKey = goes
 tokenSecret = here
+
+[confluenceSetup]
+host = confluence
+user = None
+password = None
+
+[gmailSetup]
+user = email address
```

Comment:
Where's the gmail API stuff, is it the JSON file that Google gives you from the developer site?  If so it might be good to specify that filename here, assuming it's in the same directory as this config file will be at least.

Reply:
Templates added.